### PR TITLE
doc: Add documentation about polkit rules and systemd

### LINKED ISSUE #6882: Question: Running services with non admin user (regular user)
Hi,

I'm having trouble to start services with non admin user.
I'm giving a user the following sudo permission:
my_user ALL= NOPASSWD  : /bin/systemctl start my_service

From the terminal it works well but when i'm trying to start it from cockpit services
i'm getting Error: permission denied

What am i missing here?
If i give it sudo to all or adding it to wheel group it works well but I don't want
to give him that much permissions.

Please advise,
Or

### PR DESCRIPTION
**Pull request by stefwalter**
Add documentation about how to use polkit rules to customize
systemd's privilege escalation.

Fixes #6882

### CODE REVIEW

File: doc/guide/privileges.xml [hunk 1]
```diff
@@ -39,4 +39,38 @@ $ pkexec cockpit-bridge
     user to be able to escalate privileges in this way. However both Policy Kit and
     sudo may be configured to use other criteria.</para>
 
+  <section id="privileges-polkit">
+    <title>Customizing Polkit Privileges</title>
+
+    <para>Services like <ulink url="http://www.freedesktop.org/wiki/Software/systemd/">systemd</ulink>
+      and <ulink url="https://wiki.gnome.org/Projects/NetworkManager">NetworkManager</ulink> use
+      <ulink url="http://www.freedesktop.org/wiki/Software/polkit/">Policy Kit</ulink> to
```
Comment (martinpitt) [inline doc/guide/privileges.xml]: The legacy name is "PolicyKit", the current name is "polkit"; I've never seen "Policy Kit".

File: doc/guide/privileges.xml [hunk 2]
```diff
@@ -39,4 +39,38 @@ $ pkexec cockpit-bridge
...
+      validate and escalate privileges. It is possible to customize these rules with files
+      in <filename>/etc/polkit-1/rules.d</filename>.</para>
+
+    <para>Policy Kit rules files are
+      <ulink url="https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html">javascript with specific methods and objects</ulink>. For example, placing the following polkit rule to
+      <filename>/etc/polkit-1.rules.d/10-operators.rule</filename> allows all users in the
+      <code>operators</code> group start, stop, restart and otherwise manage systemd services:</para>
```
Comment (martinpitt) [inline doc/guide/privileges.xml]: "to"  start, stop...

File: doc/guide/privileges.xml [hunk 3]
```diff
@@ -39,4 +39,38 @@ $ pkexec cockpit-bridge
...
+  <section id="privileges-polkit">
+    <title>Customizing Polkit Privileges</title>
+
+    <para>Services like <ulink url="http://www.freedesktop.org/wiki/Software/systemd/">systemd</ulink>
+      and <ulink url="https://wiki.gnome.org/Projects/NetworkManager">NetworkManager</ulink> use
+      <ulink url="http://www.freedesktop.org/wiki/Software/polkit/">Policy Kit</ulink> to
+      validate and escalate privileges. It is possible to customize these rules with files
+      in <filename>/etc/polkit-1/rules.d</filename>.</para>
+
+    <para>Policy Kit rules files are
```
Comment (martinpitt) [inline doc/guide/privileges.xml]: same here

File: doc/guide/privileges.xml [hunk 4]
```diff
@@ -39,4 +39,38 @@ $ pkexec cockpit-bridge
...
+polkit.addRule(function(action, subject) {
+    if (action.id == "org.freedesktop.systemd1.manage-units") {
+        if (subject.isInGroup("operators")) {
+            return polkit.Result.YES;
+        }
+    }
+});
+</programlisting>
+
+    <para>In order to allow in a certain group to perform any administrative action you could add
```
Comment (martinpitt) [inline doc/guide/privileges.xml]: s/in//

Comment (stefwalter) [inline doc/guide/privileges.xml]: Fixed.

File: doc/guide/privileges.xml (hunk 2 above)
Comment (stefwalter) [inline doc/guide/privileges.xml]: Fixed.

File: doc/guide/privileges.xml (hunk 3 above)
Comment (stefwalter) [inline doc/guide/privileges.xml]: Fixed.

File: doc/guide/privileges.xml (hunk 1 above)
Comment (stefwalter) [inline doc/guide/privileges.xml]: Fixed.

**Pull request merged**